### ISSUE TITLE
Unable to remap keys in Mail app

### ISSUE THREAD

**Issue by sj7trunks** [wyne/awesome-hammerspoon#1]
I can confirm that the key gets remapped as it doesn't cause the select all function... however it doesn't remap to anything afterwards. Expected result would be to archive.

```
hs.loadSpoon("AppBindings")

spoon.AppBindings:bind('Mail', {
    --  |----FROM----| |------TO------|
    --  |meta  ,  key| |meta   ,  key |
      { {'cmd'}, 'a', {'ctrl', 'cmd'}, 'a' }, -- Archive instead of select all
    })

```
   
Log:
2022-05-31 11:57:50: -- Loading Spoon: AppBindings
2022-05-31 11:57:50:          AppBinding:     Found binding for app Mail
2022-05-31 11:57:50:                          Mapping table: 0x600003ff39c0a -> table: 0x600003ff0bc0a
2022-05-31 11:57:50: -- Done.
2022-05-31 11:58:01: 11:58:01     hotkey: Enabled hotkey ⌘A
2022-05-31 11:58:03: -- Loading extension: keycodes

**Comment by wyne** [comment]
You're right, I was able to reproduce this. I'm not really sure what's going on there.

And to confirm, the remapping works for other apps, correct?

**Comment by sj7trunks** [comment]
That's correct - sorry missed this.
